lib/core/plugin/pluginContext.js

```diff
@@ -175,6 +175,34 @@ class PluginContext {
         }
       });
 
+      Reflect.defineProperty(this.accessors, 'subscription', {
+        enumerable: true,
+        value: {
+          register: (connectionId, index, collection, filters) => {
+            const request = new Request(
+              {
+                action: 'subscribe',
+                body: filters,
+                collection,
+                controller: 'realtime',
+                index,
+              },
+              {
+                connectionId: connectionId,
+              });
+            return kuzzle.ask(
+              'core:realtime:subscribe',
+              request
+            );
+          },
+          unregister: (connectionId, roomId, notify) => 
```

Comment:
According to this PR documentation, shouldn't this be the following instead?

```suggestion
      Reflect.defineProperty(this.accessors, 'realtime', {
        enumerable: true,
        value: {
          subscribe: (connectionId, index, collection, filters) => {
            const request = new Request(
              {
                action: 'subscribe',
                body: filters,
                collection,
                controller: 'realtime',
                index,
              },
              {
                connectionId: connectionId,
              });
            return kuzzle.ask(
              'core:realtime:subscribe',
              request
            );
          },
          unsubscribe: (connectionId, roomId, notify) => 
```